# Analyzing the covid trends in Delaware (02/10/2022 - 02/17/2022)

In [1]:
#importing the necessary libraries
import pandas as pd
import numpy as py

In [2]:
# reading the Covid dataset
covid = pd.read_parquet("../../../data/covid_combined_dataset.parquet.gzip")


In [3]:
covid

countyFIPS      County Name State  StateFIPS       Date  Cases  \
0              1001  Autauga County     AL          1 2020-01-22      0   
1              1001  Autauga County     AL          1 2020-01-23      0   
2              1001  Autauga County     AL          1 2020-01-24      0   
3              1001  Autauga County     AL          1 2020-01-25      0   
4              1001  Autauga County     AL          1 2020-01-26      0   
...             ...              ...   ...        ...        ...    ...   
2381631       56045   Weston County     WY         56 2022-02-13   1524   
2381632       56045   Weston County     WY         56 2022-02-14   1524   
2381633       56045   Weston County     WY         56 2022-02-15   1528   
2381634       56045   Weston County     WY         56 2022-02-16   1528   
2381635       56045   Weston County     WY         56 2022-02-17   1528   

         Death  population  
0            0       55869  
1            0       55869  
2            0       55869  
3            0       55869  
4            0       55869  
...        ...         ...  
2381631     17        6927  
2381632     17        6927  
2381633     17        6927  
2381634     17        6927  
2381635     17        6927  

[2381636 rows x 8 columns]

In [4]:
Delaware = covid[covid['State'] == 'DE']

In [5]:
Delaware

countyFIPS     County Name State  StateFIPS       Date  Cases  Death  \
239528       10001    Kent County     DE         10 2020-01-22      0      0   
239529       10001    Kent County     DE         10 2020-01-23      0      0   
239530       10001    Kent County     DE         10 2020-01-24      0      0   
239531       10001    Kent County     DE         10 2020-01-25      0      0   
239532       10001    Kent County     DE         10 2020-01-26      0      0   
...            ...             ...   ...        ...        ...    ...    ...   
241797       10005  Sussex County     DE         10 2022-02-13  59881    772   
241798       10005  Sussex County     DE         10 2022-02-14  60061    778   
241799       10005  Sussex County     DE         10 2022-02-15  60089    778   
241800       10005  Sussex County     DE         10 2022-02-16  60149    784   
241801       10005  Sussex County     DE         10 2022-02-17  60232    784   

        population  
239528      180786  
239529      180786  
239530      180786  
239531      180786  
239532      180786  
...            ...  
241797      234225  
241798      234225  
241799      234225  
241800      234225  
241801      234225  

[2274 rows x 8 columns]

In [6]:
#Taking the dates from 10th feb 2020
Status_DE = Delaware.loc[(Delaware["Date"] > "2022-02-09")]

In [7]:
Status_DE.head()

countyFIPS   County Name State  StateFIPS       Date  Cases  Death  \
240278       10001  Kent County     DE         10 2022-02-10  48751    576   
240279       10001  Kent County     DE         10 2022-02-11  48839    576   
240280       10001  Kent County     DE         10 2022-02-12  48839    576   
240281       10001  Kent County     DE         10 2022-02-13  48839    576   
240282       10001  Kent County     DE         10 2022-02-14  48999    587   

        population  
240278      180786  
240279      180786  
240280      180786  
240281      180786  
240282      180786

In [8]:
# Dropping 'countyFIPS','StateFIPS'
Status_DE = Status_DE.drop(['countyFIPS','StateFIPS'], axis=1)

In [9]:
Status_DE

County Name State       Date   Cases  Death  population
240278        Kent County     DE 2022-02-10   48751    576      180786
240279        Kent County     DE 2022-02-11   48839    576      180786
240280        Kent County     DE 2022-02-12   48839    576      180786
240281        Kent County     DE 2022-02-13   48839    576      180786
240282        Kent County     DE 2022-02-14   48999    587      180786
240283        Kent County     DE 2022-02-15   49030    587      180786
240284        Kent County     DE 2022-02-16   49092    589      180786
240285        Kent County     DE 2022-02-17   49165    589      180786
241036  New Castle County     DE 2022-02-10  143136   1253      558753
241037  New Castle County     DE 2022-02-11  143381   1253      558753
241038  New Castle County     DE 2022-02-12  143381   1253      558753
241039  New Castle County     DE 2022-02-13  143381   1253      558753
241040  New Castle County     DE 2022-02-14  143779   1278      558753
241041  New Castle County     DE 2022-02-15  143852   1280      558753
241042  New Castle County     DE 2022-02-16  143931   1281      558753
241043  New Castle County     DE 2022-02-17  144131   1282      558753
241794      Sussex County     DE 2022-02-10   59772    772      234225
241795      Sussex County     DE 2022-02-11   59881    772      234225
241796      Sussex County     DE 2022-02-12   59881    772      234225
241797      Sussex County     DE 2022-02-13   59881    772      234225
241798      Sussex County     DE 2022-02-14   60061    778      234225
241799      Sussex County     DE 2022-02-15   60089    778      234225
241800      Sussex County     DE 2022-02-16   60149    784      234225
241801      Sussex County     DE 2022-02-17   60232    784      234225

# Observation

 We are analyzing the Covid trends in Delaware form 10-feb-2022 to 17th feb 2022

## Cases

In [10]:
# maximum number of Cases
Status_DE.Cases.max()

144131

In [11]:
# minimum number of Cases
Status_DE.Cases.min()

48751

In [12]:
# Average number of Cases
Status_DE.Cases.mean()

84186.33333333333

* As for the number of cases reported in Delware, the highest number of cases were reported on Nw castle county on 17th feb 2022 with '144131' cases

* Minimum were '48751' cases in Kent County on 10th feb 2022

* average reported cases between Feb 10th - Feb 17th 2022 were '84186'

## Deaths

In [13]:
# maximum number of Deaths
Status_DE.Death.max()

1282

In [14]:
# minimum number of Deaths
Status_DE.Death.min()

576

In [15]:
# Average number of Deaths
Status_DE.Death.mean()

875.0416666666666

* As for the number of deaths reported in Delware, the highest number of deaths were reported on Nw castle county on 17th feb 2022 with '1282' deaths

* Minimum were '576' deaths in Kent County on 10th feb 2022

* average reported deaths between Feb 10th - Feb 17th 2022 were '875'

## Conclusion

As for conclusion both the cases and deaths has been slightly Increased this week